website/docs/guides/upgrade.html.markdown

```diff
@@ -109,6 +109,33 @@ consumed using the data sources `nsxt_edge_upgrade_group` and `nsxt_host_upgrade
 **NOTE:** Only one `nsxt_upgrade_run` resource should be used in the upgrade plan.
 
 ```hcl
+data "nsxt_policy_host_transport_node" "host_transport_node1" {
+  display_name = "hosttransportnode1"
+}
+
+data "nsxt_policy_host_transport_node" "host_transport_node2" {
+  display_name = "hosttransportnode2"
+}
+
+data "nsxt_policy_host_transport_node" "host_transport_node3" {
+  display_name = "hosttransportnode3"
+}
+
+data "nsxt_host_upgrade_group" "hg1" {
+  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
+  display_name       = "Group for Compute-Cluster1"
+}
+
+data "nsxt_edge_upgrade_group" "eg1" {
+  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
+  display_name       = "edgeupgradegroup1"
+}
+
+data "nsxt_edge_upgrade_group" "eg2" {
+  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
+  display_name       = "edgeupgradegroup2"
+}
+
```

Comment:
Would it be more helpful to use `display_name`s that are in camelCase or simila for readability? For example:

```hcl
data "nsxt_policy_host_transport_node" "host_transport_node1" {
  display_name = "hostTransportNode1"
}

data "nsxt_policy_host_transport_node" "host_transport_node2" {
  display_name = "hostTransportNode2"
}

data "nsxt_policy_host_transport_node" "host_transport_node3" {
  display_name = "hostTransportNode3"
}

data "nsxt_host_upgrade_group" "hg1" {
  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
  display_name       = "hostUpgradeGroup1"
}

data "nsxt_edge_upgrade_group" "eg1" {
  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
  display_name       = "edgeUpgradeGroup1"
}

data "nsxt_edge_upgrade_group" "eg2" {
  upgrade_prepare_id = nsxt_upgrade_prepare.test.id
  display_name       = "edgeUpgradeGroup2"
}
```

Reply:
OK. Note that group names are set by NSX - I've corrected the edge group to have the format which is used by NSX as well (edgegroup-<edge_cluster_name>).